src/ribasim_nl/ribasim_nl/cloud.py

```diff
@@ -0,0 +1,282 @@
+import logging
+import os
+import shutil
+from dataclasses import dataclass, field
+from pathlib import Path
+from typing import List, Tuple, Union
+from xml.etree import ElementTree
+
+import requests
+
+logger = logging.getLogger(__name__)
+
+RIBASIM_NL_CLOUD_PASS = os.getenv("RIBASIM_NL_CLOUD_PASS")
+RIBASIM_NL_DATA_DIR = os.getenv("RIBASIM_NL_DATA_DIR")
+RIBASIM_NL_CLOUD_USER = "nhi_api"
+WEBDAV_URL = "https://deltares.thegood.cloud/remote.php/dav"
+BASE_URL = rf"{WEBDAV_URL}/files/{RIBASIM_NL_CLOUD_USER}/Ribasim modeldata"
+
+WATER_AUTHORITIES = [
+    "AaenMaas",
+    "AmstelGooienVecht",
+    "BrabantseDelta",
+    "DeDommel",
+    "Delfland",
+    "DrentsOverijsselseDelta",
+    "HollandseDelta",
+    "HollandsNoorderkwartier",
+    "HunzeenAas",
+    "Limburg",
+    "Noorderzijlvest",
+    "RijnenIJssel",
+    "Rijnland",
+    "Rivierenland",
+    "Rijkswaterstaat",
+    "Scheldestromen",
+    "SchielandendeKrimpenerwaard",
+    "StichtseRijnlanden",
+    "ValleienVeluwe",
+    "Vechtstromen",
+    "WetterskipFryslan",
+    "Zuiderzeeland",
+]  # noqa
+
+HIDDEN_DIRS = ["D-HYDRO modeldata"]  # somehow this dir-name still exists :-(
+
+
+def is_dir(item):
+    return Path(item).suffix == ""
```

Comment:
Why not use `Path.is_dir()`? I don't think we need this function.

Reply:
Is used in https://github.com/Deltares/Ribasim-NL/blob/a3aa8b8f55fac0276372cd9546c38799e37368a1/src/ribasim_nl/ribasim_nl/cloud.py#L250. `Path("my_dir").is_dir()` is only True if path is directory ánd the directory exists. If you have better code, please provide.

Reply:
Ah ok it wasn't clear to me that this should be true if the directory doesn't exist. Would be good to clarify in a docstring.